Libs/GLib/Classes/Exception.IDisposable.cs

```diff
@@ -1,30 +1,24 @@
 using System;
+using System.Runtime.InteropServices;
 
 namespace GLib
 {
-    public partial class GException : IDisposable
+    public partial class GException
     {
-        #region IDisposable Implementation
-
-        private bool _disposedValue;
-
-        ~GException() => Dispose(false);
-
-        public void Dispose()
+        private class GExceptionSafeHandle : SafeHandle
```

Comment:
This file should be renamed from `Exception.IDisposable.cs` to `Exception.SafeHandle.cs`.